### DISCUSSION TITLE
How can I clean project after build the CodeChecker?

### DISCUSSION THREAD

**Discussion by VladAdGad** [Ericsson/codechecker#3548] [Q&A] [answered]
I actually use a docker to setup your really awesome project. And I recognize that after build size of it is 500mb. SO i decided to remove everything except folder build. And I met a lot of problems such as no module found etc. So how can I clean unnecessary stuff after build?

My command looks like
```
RUN git clone --depth 1 --branch v6.15.0 https://github.com/Ericsson/codechecker.git /opt/codechecker \
    && cd /opt/codechecker \
    && make venv \
    && . ./venv/bin/activate \
    && make package \
    && ls --hide=build | xargs --delimiter="\n" rm --recursive --force
```

**Comment by VladAdGad** [comment]
Much better now with `BUILD_LOGGER_64_BIT_ONLY=YES BUILD_UI_DIST=NO make package`

**Comment by csordasmarton** [comment, marked as answer]
If you are using a virtual env you will be able to build the package but you will not be able to run a container because the venv will be activated only during the build process and not when the container will be created.

We already have a docker image for the web part: https://github.com/Ericsson/codechecker/blob/master/web/docker/Dockerfile
You can check it how we build the CodeChecker.

Instead of creating a virtual env you should install the python requirements in the following way:
https://github.com/Ericsson/codechecker/blob/75ef0ec24f9ef0f3ae6cd0e3fc57d04ab1114c21/web/docker/Dockerfile#L76-L101

Also If you don't want to build the GUI part you can skip to install those requirements.
